VK/VK/Services/Models/NewsPostModels/NewsPostResponse.swift

```diff
@@ -0,0 +1,16 @@
+// NewsPostResponse.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Новости
+struct NewsPostResponse: Decodable {
+    let response: Response
+}
+
+/// Ответ с новостями
+struct Response: Decodable {
+    let count: Int
```

Comment:
```suggestion
    let count: Int
```
добавь описание всем пропертям